xbmc/addons/interfaces/AudioDSP/AddonCallbacksAudioDSP.cpp

```diff
@@ -169,7 +170,7 @@ ADSPHANDLE CAddonCallbacksADSP::ADSPSoundPlay_GetHandle(void *addonData, const c
     return NULL;
   }
 
-  IAESound *sound = CAEFactory::MakeSound(filename);
+  IAESound *sound = CServiceBroker::GetActiveAE().MakeSound(filename);
```

Comment:
Is the method required to delete it or is that managed elsewhere is what I meant by owning pointer. Raw pointers are fine if they're not owning the resource, should be avoided if they're owning and instead use unique/shared_ptr